cmd/config/internal/commands/cmdwrap.go

```diff
@@ -106,7 +109,11 @@ func (r *WrapRunner) runE(c *cobra.Command, args []string) error {
 		inputs = append(inputs, &kio.ByteReader{Reader: mergeInput})
 		fltrs = append(fltrs, &filters.MergeFilter{})
 	}
-	inputs = append(inputs, &kio.ByteReader{Reader: xargsOut})
+	out := strings.TrimPrefix(xargsOut.String(), `Command "xargs" is deprecated, this will no longer be available in kustomize v5.
```

Comment:
This needs to be done because stdout (which contains the deprecation message) is being sent to the filters, which are trying to parse it as a KRM resource.

Reply:
Hmm, that suggests to me that our deprecation is actually a breaking change for the `xargs` command, in that users would also need to filter this message out of the output. I wonder if that's true of any other deprecations. IMO these messages should be going to stderr and the "consumable" output should be going to stdout. Looks like `Command.SetOut` might do the trick? https://github.com/spf13/cobra/blob/de187e874d1ca382320088f8f6d76333408e5c2e/command.go#L247-L251

Reply:
`Command.SetOut(os.Stderr)` resulted in everything, including the command's "consumable" output to `stdout`. I instead added a `preRunE` function to print the deprecation message to stderr. WDYT?

Reply:
Approach LGTM. Just wondering if we can revert the test changes that previously required an exact match and now use `Contains` (and if not, why).

Reply:
Reverted the change to make it `Contains`